inc/scroom/transformpresentation.hh

```diff
@@ -60,7 +60,12 @@ public:
   virtual std::string getTitle();
 
   // PipetteViewInterface
-  virtual PipetteLayerOperations::PipetteColor getAverages(Scroom::Utils::Rectangle<int> area);
+  /** 
+   * Returns the average pixel values for each component, contained in the area
+   * 
+   * @param area selected area to get the pixel values from.
+   */
+  virtual PipetteLayerOperations::PipetteColor getPixelAverages(Scroom::Utils::Rectangle<int> area);
```

Comment:
When you implement an interface, do not repeat the documentation.

I think, doxygen is smart enough to look up the documentation of the base class. And even if not, repeating the same comment over and over in the code is begging for trouble if they ever need to be updated.